sphinx/texinputs/Makefile_t

```diff
@@ -82,11 +87,14 @@ xz: tar
 {%- endif %}
 	$(PDFLATEX) $(LATEXOPTS) "$<"
 
+{%- if latex_engine != 'xelatex' %}
+
 %.ps: %.dvi
-	dvips '$<'
+	dvips "$<"
+{%- endif %}
 
 clean:
-	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI) *.fls *.fdb_latexmk
+	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI) $(ALLXDV) *.fls *.fdb_latexmk
```

Comment:
Actually, I invoke `make clean` in `_build/latex`, I get this result:
```
(venv) C:\Project\sphinx-dev\issues\3082\_build\latex>make clean
rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz 3082.pdf 3082.dvi *.fls *.fdb_latexmk
process_begin: CreateProcess(NULL, rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz 3082.pdf 3082.dvi *.fls *.fdb_latexmk, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [setup] Error 2
```
I think this is a different new 'gmake on windows' problem...